Fix a typo in opcache func info for "get_parrent_class"

TysonAndre opened the pull request:
This should only have one r.
http://php.net/get_parent_class mentions that it returns string|false

Also, I'm not 100% sure if this is right - should this include
MAY_BE_NULL to handle the case of too many parameters?
(then again, get_called_class() also returns null with too many parameters)

```
php > var_export(get_parent_class(2,3));
Warning: get_parent_class() expects at most 1 parameter, 2 given in php
shell code on line 1
NULL
```

nikic commented:
> Also, I'm not 100% sure if this is right - should this include
MAY_BE_NULL to handle the case of too many parameters?

Yes, on 7.4 it needs to include MAY_BE_NULL. In master it's no longer necessary.

> (then again, get_called_class() also returns null with too many parameters)

Functions that accept no arguments are treated differently in this code.

TysonAndre commented:
> Yes, on 7.4 it needs to include MAY_BE_NULL. In master it's no longer necessary.

Fixed and changed to `	FN("get_parent_class",        MAY_BE_FALSE | MAY_BE_STRING | MAY_BE_NULL),`

krakjoe commented:
Merged as 98b487e244a81fa02d1ca98b06657e3d88dced02

Thanks.
